docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+\end{itemize}
+
+\section{Verification of \texttt{pyuvdata} phasing methods}\label{appx:verify}
+As part of standard unit testing, astrometry calculations are tested between the three primary software packages (NOVAS, ERFA, astropy), to check for consistency. Standard tests use a small range of dates starting with May 11, 2014 (JD 2456789.0), at ICRS coordinates $\alpha$=09h25m37.43612s $\delta$=+40d52m10.7709s. These values were chosen somewhat pseudo-randomly, although more detailed comparison with additional datasets (as discussed below) has verified that the astrometry is valid outside of just this particular position and time. ERFA and astropy are generally agree to within a few microarcseconds. NOVAS shows larger magnitudes of disagreement (of order 0.2 mas), although neglecting celestial pole offsets (which neither astropy or pyerfa currently account for) and accounting for differences in calculated LST, these differences shrink to of order 10s of microarcseconds. Unit testing for \texttt{pyuvdata} requires that these methods produce a consistent set of results to within 1 mas.
+
+Astrometry calculations have also been compared against a select number of test data sets, including those from ATCA and CARMA (\verb!atca_miriad! and \verb!carma_miriad!), which contain apparent coordinates as part of their metadata. Astrometry appears to agree down to a level of approximately 200 mas, which is a larger than expected, though anecdotal reports indicate that these data sets may not have diurnal aberration accounted for, which is consistent with the discrepancy. Calculations have also been done against several SMA data sets, which uses the NOVAS library as part of real time calculations. Values from these data sets agree with \texttt{pyuvdata} calculated values to of order 1 mas.\footnote{An older test data set located in the repository shows discrepancies of order 200 mas, although was traced back to an incorrect application of polar wobble corrections.}
+
+$uvw$-coordinates have been similarly checked against an MWA data sets -- a detailed comparison can be found here \href{https://github.com/RadioAstronomySoftwareGroup/pyuvdata/files/6525743/new.phasing.pdf}{here}, but the calculated positions agree with the MWA Cotter\footnote{\url{https://github.com/MWATelescope/cotter}} derived positions to better than a part in $10^7$ (after accounting for the apparent lack of frame position angle rotation in the latter). Additional testing with SMA datasets show a similar level of agreement, with the primary limitation arising from minor errors in the recorded LST versus something intrinsic to the calculation itself. 
```

Comment:
an MWA data sets -> an MWA data set

Also the reference isn't getting formatted properly. Could be a footnote if needed.

Reply:
The last sentence here belongs in the prior paragraph I think.

Reply:
Okay, fixed, made into a footnote.